### ISSUE TITLE
Feature Request: Is it possible to add offline support 

### ISSUE THREAD

**Issue by MBcom** [haiwen/seadroid#649]
Could you add an offline support to the app, so you can open and edit files without a server connection. In addition, it would be nice if a file Upload was possible without a connection to the server with the app synchronizing them automatically as soon as a connection is possible again.
Thank you in advance

**Comment by shoeper** [comment]
Duplicate of https://github.com/haiwen/seadroid/issues/4 and https://github.com/haiwen/seadroid/issues/55

**Comment by DoS007** [comment]
I don't think so. Auto upload is no problem nowadays because seafile detects file changes and uploads automatically.  This delayed upload may be a problem. Maybe use an upload queue for this?

What is really interesting in my opinion is that some files are always updated (downloaded) when changes happen. E.g. you use office word files. Then you want to have always the newest versions. But starting word out of seafile does not work as word then doesn't allow to save this file there. So you have to start the file out of word itself. But then you won't have nesseccarily the updated version at this moment, because you can't declare a file to be autoupdated.

**Comment by zhwanng** [comment]
Offline mode has been added to Seafile Android, you can view the data or files that have been cached, but it is best not to edit/back up any files while offline.
